package.json

```diff
@@ -30,18 +30,14 @@
     "jest": "^27.5.1",
     "lerna": "^5.5.1",
     "lint-staged": "^13.0.3",
-    "prettier": "^2.7.1",
-    "tslint": "^6.1.3",
-    "tslint-config-prettier": "^1.18.0",
-    "tslint-config-standard": "^9.0.0",
-    "tslint-react": "^5.0.0"
+    "prettier": "^2.7.1"
   },
   "lint-staged": {
     "*.{js,json,jsx,css,md}": [
       "prettier --write"
     ],
     "*.{ts,tsx}": [
-      "tslint --fix",
+      "eslint --fix",
```

Comment:
When commit, I got this error:
<img width="804" alt="Screenshot 2022-09-26 at 14 17 17" src="https://user-images.githubusercontent.com/31354481/192263250-75aee2e9-76f1-49c8-8ca9-85ced444fccd.png">

Reply:
Just noticed same error. I'll try to fix that.